pkg/install/gatherlogs.go

```diff
@@ -32,8 +36,16 @@ func (i *Installer) gatherFailureLogs(ctx context.Context) {
 			continue
 		}
 
-		i.log.Printf("%s: %s", runtime.FuncForPC(reflect.ValueOf(f).Pointer()).Name(), string(b))
+		i.log.Printf("%s: %s", log.FriendlyName(logFunc), string(b))
+	}
+
+	if _, ok := origErr.(*api.CloudError); !ok {
+		err = fmt.Errorf("%s: %s", f.Name(), origErr)
+	} else {
+		err = origErr
 	}
+	return err
```

Comment:
You could just return the value you are currently assigning to `err`:

```suggestion
	if _, ok := origErr.(*api.CloudError); !ok {
		return fmt.Errorf("%s: %s", f.Name(), origErr)
	}
	return origErr
```